packages/api-client/src/granules.ts

```diff
@@ -48,12 +57,7 @@ export const getGranuleResponse = async (params: {
     callback = invokeApi,
   } = params;
 
-  let path = `/granules/${collectionId}/${granuleId}`;
-
-  // Fetching a granule without a collectionId is supported but deprecated
-  if (!collectionId) {
-    path = `/granules/${granuleId}`;
-  }
+  const path = encodeGranulesURIComponent(granuleId, collectionId);
```

Comment:
So I would hope not because the caller of this function isn't passing the path but rather the explicit collection and granule IDs. I'd say it's up to the caller to recognize that and not try to do any fancy encoding before hand. A quick search shows we're not encoding prior to calling this function so I think we're ok - assuming you're cool with my thought process.

Reply:
Yup, absolutely. That seemed like the most likely scenario.